README.md

```diff
@@ -1,143 +1,110 @@
-# Conduit PostgreSQL Connectors 
+# Conduit Connector PostgreSQL
 
 # Source
-The Postgres Source Connector connects to a database with the provided `url` and
-then will call `Ping` to test the connection. If the `Ping` fails, the `Open`
-method will fail and the Connector won't be started.
 
-Upon starting, the source takes a snapshot of a given table in the database, 
-then switches into CDC mode. In CDC mode, the plugin reads from a buffer of 
-CDC events.
+The Postgres Source Connector connects to a database with the provided `url` and starts creating records for each change
+detected in a table.
+
+Upon starting, the source takes a snapshot of a given table in the database, then switches into CDC mode. In CDC mode,
+the plugin reads from a buffer of CDC events.
 
 ## Snapshot Capture
-When the connector first starts, snapshot mode is enabled. The connector
-acquires a read-only lock on the table, and then reads all rows of the table 
-into Conduit. Once all of the rows in that initial snapshot are read, then the 
-connector releases its lock and switches into CDC mode. 
 
-This behavior is enabled by default, but can be turned off by adding 
-`"snapshot":"off"` to the Source configuration.
+When the connector first starts, snapshot mode is enabled. The connector acquires a read-only lock on the table, and
+then reads all rows of the table into Conduit. Once all rows in that initial snapshot are read the connector releases
+its lock and switches into CDC mode.
+
+This behavior is enabled by default, but can be turned off by adding `"snapshotMode":"never"` to the Source
+configuration.
```

Comment:
Nitpick: it appears that we're using different line lengths in our IDEs. It would be nice to agree on one and use it (otherwise we might see noise in future PRs).